Classes/Aggregate/NewContentElementWizardAggregate.php

```diff
@@ -113,11 +148,18 @@ protected function processElement(array $element)
             (!empty($element['description'])) ? $element['description'] : ''
         );
 
+        $extensionConfiguration = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['mask_export']);
+        if (!empty($extensionConfiguration['exportIcons'])) {
+            $iconName = substr($element['icon'], 3);
+        } else {
+            $iconName = 'content-textpic';
```

Comment:
Introduce and use $iconIdentifier = $iconName !== '' ? $iconName : 'content-textpic';